### ISSUE TITLE
On Kotlin versions at or above 1.7.0, cocoapods-generate is still required

### ISSUE THREAD

**Issue by LandryNorris** [Kotlin/kdoctor#27]
After Kotlin 1.7.0, the cocoapods-generate tool is no longer required. I still get the following:

```
* cocoapods-generate plugin not found
          Get cocoapods-generate from https://github.com/square/cocoapods-generate#installation
    
Failures: 1
KDoctor has diagnosed one or more problems while checking your environment.
Please check the output for problem description and possible solutions.
```

Should this be included in the output still? It's useful information if the user is below Kotlin 1.7.0, but not useful for users using Kotlin 1.7.0 or higher.

**Comment by LandryNorris** [comment]
This could potentially be demoted to a warning with a note about the language version, could it not?

**Comment by terrakok** [comment]
Fixed in 0.0.5
